po/es.po

```diff
@@ -156,53 +155,48 @@ msgstr "Día 1: Tarde"
 #: src/SUMMARY.md src/tuples-and-arrays.md
 #: src/tuples-and-arrays/tuples-and-arrays.md
 msgid "Tuples and Arrays"
-msgstr ""
+msgstr "Tuplas y Matrices"
```

Comment:
Matriz implies 2D array or higher. Isn't there a better translation for "Array"?

Reply:
What do you think about using "Arreglos"?

Reply:
Arreglos is fine.

Reply:
Perfect!